contracts/MediumPriceOracle.sol

```diff
@@ -0,0 +1,79 @@
+pragma solidity 0.4.24;
+
+import "zeppelin-solidity/contracts/ownership/Ownable.sol";
+import "zeppelin-solidity/contracts/math/SafeMath.sol";
+import "./OracleCasting.sol";
+import "./interfaces/ISignedOracle.sol";
+
+
+/**
+ * @title UintPluracle
+ * @dev An oracle of uint256 types oracles that provides a medium price.
+ */
+contract UintPluracle is Ownable {
```

Comment:
Should we make this consistent with the title of the file?

Reply:
yes, changing it !